
<launch>
    <node pkg="ros_pololu_servo" type="ros_pololu_servo_node" name="ros_pololu_servo_node" output="screen">
        <param name="pololu_config" value="$(find ros_pololu_servo)/launch/pololu.yaml" />
        <param name="port_name" value="/dev/ttyACM0" />
        <param name="baud_rate" value="115200" />
        <param name="channel_min_value" value="4000" />
        <param name="channel_max_value" value="8000" />
        <param name="rate_hz" value="10" />
    </node>
</launch>
